mmv1/third_party/terraform/provider/provider_mmv1_resources.go.tmpl

```diff
@@ -190,11 +191,9 @@ var handwrittenDatasources = map[string]*schema.Resource{
 	"google_parameter_manager_parameter":               parametermanager.DataSourceParameterManagerParameter(),
 	"google_parameter_manager_parameters":              parametermanager.DataSourceParameterManagerParameters(),
 	"google_parameter_manager_parameter_version":       parametermanager.DataSourceParameterManagerParameterVersion(),
-	"google_parameter_manager_parameter_version_render":parametermanager.DataSourceParameterManagerParameterVersionRender(),
```

Comment:
Seems like we shouldn't include the removal of these datasources, at least not as part of this PR.